### ISSUE TITLE
Opening / refreshing Mercury page automatically downloads `html` file 

### ISSUE THREAD

**Issue by jvivian** [mljar/mercury#34]
Hi @pplonski , thank you for the wonderful tool.

I built a test page and noticed that every time the page loads (in either chrome or edge) it downloads a file called `html` which contains the following:
```
<style type="text/css">
.jp-mod-noOutputs {
    padding: 0px; 
}
.jp-mod-noInput {
  padding-top: 0px;
  padding-bottom: 0px;
}
</style>
```
![image](https://user-images.githubusercontent.com/7576256/152648650-b0c10385-26b6-4112-9ca4-860798f02876.png)

Happy to help debug or test if you cannot reproduce on your end!

I am using version: 0.3.0

Cheers,
John

**Comment by pplonski** [comment]
Hi @jvivian,

Thank you for reporting! It is custom css part that I've added to escape padding from code cells that are hidden. This css should be in the HTML file that is displayed in the website. Does your HTML file contain only css part? 

Thank you for your help!

**Comment by jvivian** [comment]
Hi @pplonski ,

Travelling at the moment so excuse the delay. I also wanted to check to see if the problem occurred on a different machine/OS. 

> Does your HTML file contain only css part?

Yes — the file that is automatically downloaded only contains the `<style>` block I originally included.

I looked at the source of the page that Mercury displays and under `Styles` I don't see that padding (or in the raw HTML) but the spacing looks good and I have code hidden:

![image](https://user-images.githubusercontent.com/7576256/153045560-81b3d94f-4549-4675-9142-f90d617ca359.png)

**Comment by jumpei-tsutsui** [comment]
Hi, thank you for your great work!

Same issue here.
Although I have not investigate the details yet, it seems that a file named `/media/.html` is downloaded in this case.

The environment in which the issue occurred is as follows:
- Python Official Container Image (Tag: 3.10)
  - https://hub.docker.com/_/python
- Docker Desktop on MacOS
- Pipenv
- Chrome v100

Here is the version information and the process running on the container.
```
root@mercury-app:/app# pipenv run python --version
Python 3.10.4
root@mercury-app:/app# pipenv requirements | grep mercury
mljar-mercury==0.6.6
root@mercury-app:/app# ps ax
  PID TTY      STAT   TIME COMMAND
    1 ?        Ss     0:02 /app/.venv/bin/python /app/.venv/bin/mercury run 0.0.0.0:8000
   11 ?        S      0:06 /app/.venv/bin/python /app/.venv/bin/celery -A mercury.server worker --loglevel=error -P gevent --concurrency 1 -E
   20 pts/0    Ss     0:00 bash
  354 pts/0    R+     0:00 ps ax
```

The same problem also occurred when I ran the container on GCP's Cloud Run and AWS's App Runner.

When I run Mercury on MacOS directly with the same command, the appropriate URL (such as `http://localhost:8000/media/-25379.html` ) appears in the Chrome Dev Tools and the file is not downloaded to my local folder.

Hope this helps you.

Thanks.

![スクリーンショット 2022-04-11 11 49 36](https://user-images.githubusercontent.com/6813186/162661383-af5e8c37-3c98-4582-8d47-4d4017d776c7.png)
![スクリーンショット 2022-04-11 11 50 46](https://user-images.githubusercontent.com/6813186/162661450-c9a6e316-42c7-4734-bfaf-26900b438765.png)

**Comment by pplonski** [comment]
@jumpei-tsutsui thank you for reporting!

hmmm, what is the `title` of the notebook in the YAML params? Could you please share the YAML config that you used? is it empty? Is it happening to all notebooks in the Mercury?

**Comment by jumpei-tsutsui** [comment]
I changed the value of the `title` to `title1` and that solved the issue! The file name is now `title1.html` .

Actually, the title was Japanese (like `タイトル` ) when the issue occurred.
I don't know how that file name was determined, because the same title had a different file name each time. The conversion may not work for some reason.

Thank you for your help!

**Comment by pplonski** [comment]
@jumpei-tsutsui thanks! I have an idea where can be a bug. Will let you know when fixed.

**Comment by pplonski** [comment]
It is fixed in the newest version of Mercury (0.6.15).
